pkg/azurefile/nodeserver.go

```diff
@@ -155,6 +168,12 @@ func (d *Driver) NodeStageVolume(ctx context.Context, req *csi.NodeStageVolumeRe
 
 	volumeID := req.GetVolumeId()
 	context := req.GetVolumeContext()
+
+	if context[serviceAccountTokenField] == "" && context[clientIDField] != "" {
```

Comment:
`clientID` is a user input parameter, since all user input parameters are not case sensitive, suggest using a func to check whether it's a token request